tests/test_models.py

```diff
@@ -0,0 +1,90 @@
+from .base import BaseTinyFatTest
+from tinyfatdb import TinyFatDB, TinyFatTable
+from tinydb import Query as Q
+
+
+########################################################################
+class TestFatModels(BaseTinyFatTest):
+    """
+    Tests to ensure custom methods on instances of TinyFatModel work,
+    both for the default table and for additional tables.
+    """
+
+    ####################################################################
+    def setUp(self):
+        super(TestFatModels, self).setUp()
+        self.db = TinyFatDB.create(name="ABC", table=TinyFatTable)
+        self.table = self.db.table(name="Foo", table=TinyFatTable)
+        self.insertable = {"a": 1, "b": 2, "c": 3}
+        self.additional_insertable = {"short": "a", "medium": "aa", "long": "aaa"}
+
+    ####################################################################
+    def insert_entry(self, container, insertable):
+        """
+        Takes a container (db, or table) and returns a Model representation
+        of the entry that was inserted.
+        """
+        
+        eid = container.insert(insertable)
+        expected_entry = {**insertable, "eid": eid}
+        return expected_entry
+
+    ####################################################################
+    def test_default_table__get(self):
+        expected_entry = self.insert_entry(self.db, self.insertable)
+        actual_entry = self.db.get(eid=expected_entry["eid"])
+        self.assertEqual(actual_entry, expected_entry)
```

Comment:
These tests are the main ones that I changed.

Previously the code was looking at the summation of the dictionary values. I felt that finding the sum of the products of the entry was not a particularly great way to determine that we got what we expected. For instance, if we had put in
``` foo = {"a:" 1, "b": 2, "c": 3}```
And we checked for the sum to be `6`, then if the following entry was returned, the tests would still pass:
``` {"a":0, "b": 5", "c": 1}```
The second entry is clearly not what was desired, but the tests would not have shown that. There also is a problem in the fact that the `TinyFatModel` will gives us back the `eid`, which is an Integer. The summation would be different depending on the value of the `eid` (which could potentially change if we were running a single test, or a suite of tests).

Because the expected return is a dictionary of values, it would be easy enough to compare the contents of the returned entry with the information that we inserted.